### DISCUSSION TITLE
Use Spotbugs Standalone

### DISCUSSION THREAD

**Discussion by ngocnn97** [spotbugs/spotbugs#2372] [Q&A] [answered]
Hello everybody, i want to use Spotbugs in mode Standalone to scan Java Project filesystem, any step-by-step instructions are greatly appreciated. 
Thanks

**Comment by iloveeclipse** [comment, marked as answer]
See https://spotbugs.readthedocs.io/en/latest/running.html

**Reply by ngocnn97** [reply]
I have read but can't use to scan files, require using GUI to create project before scanning

**Reply by ngocnn97** [reply]
do i have to scan from source or jar file
